### DISCUSSION TITLE
Availability for macos-14-arm64 in ADO

### DISCUSSION THREAD

**Discussion by Cokile** [actions/virtual-environments#9815] [General questions] [answered]
It looks like Github Actions has deployed `macos-14-arm64`:
![image](https://github.com/actions/runner-images/assets/8230448/9b2480ca-d94e-4153-a30a-dcde7869d25d)

But it's not available in Azure Pipeline pool:
![image](https://github.com/actions/runner-images/assets/8230448/c7472e29-ffbc-4335-af33-701cbfa64e12)

whether/when `macos-14-arm64` will be available in ADO?

**Comment by erik-bershel** [comment, marked as answer]
Hey @Cokile 
All the available information or plans related to this question will be published here: https://github.com/actions/runner-images/issues/8971. For now I have nothing to state about this matter.
